tools/biomodelsML/bio_ml_tool.xml

```diff
@@ -0,0 +1,89 @@
+<tool id="bio_ml_tool" name="Random Forest model" version="0.0.1" profile="22.05">
+    <description>to predict efficacy of immune checkpoint blockade across multiple cancer patient cohorts</description>
+    <requirements>
+        <container type="docker">docker.io/anupkumar/immune-checkpoint:latest</container>
```

Comment:
maybe also rename the container to fit the tool_id?

Reply:
thanks, fixed!